terraform/provider/provider.go

```diff
@@ -42,7 +42,7 @@ import (
 
 const (
 	// minServerVersion is the minimal teleport version the plugin supports.
-	minServerVersion = "6.1.0-beta.1"
+	minServerVersion = "15.0.0-0"
```

Comment:
Could we add this to a release checklist so we don't forget to bump the number in the future?

Reply:
I'll check with roman, I'm not sure where to put this but we definitely want to have this in our major release process

Reply:
Thanks.